docs/Server.md

```diff
@@ -352,7 +352,8 @@ You can change this default setting by passing the option `querystringParser` an
 ```js
 const qs = require('qs')
 const fastify = require('fastify')({
-  querystringParser: str => qs.parse(str)
+  // if caseSenstive is set to false, then parse str in lowercase
+  querystringParser: str => qs.parse(!caseSensitive ? str.toLowerCase() : str)
```

Comment:
Why this change?

Reply:
This PR points the issue #2644. There it was mentioned to change the docs when caseSensitive() is false

Reply:
Thanks for pointing it out, but where the `caseSensitive` var is defined?

Should not it be set as `fastify.initConfig.caseSensitive`?

Reply:
I have made the changes as requested and updated the PR. Thanks for guiding me out. 😃